pkg/daemon/rpm-ostree.go

```diff
@@ -275,6 +285,41 @@ func (r *RpmOstreeClient) Rebase(imgURL, osImageContentDir string) (changed bool
 	return
 }
 
+// IsBootableImage determines if the image is a bootable (new container formet) image, or a wrapper (old container format)
+func (r *RpmOstreeClient) IsBootableImage(imgURL string) (bool, error) {
+
+	// TODO(jkyros): This is duplicated-ish from Rebase(), do we still need to carry this around?
+	var isBootableImage string
+	var imageData *types.ImageInspectInfo
+	var err error
+	if imageData, err = imageInspect(imgURL); err != nil {
+		if err != nil {
+			var podmanImgData *imageInspection
+			glog.Infof("Falling back to using podman inspect")
+
+			if podmanImgData, err = podmanInspect(imgURL); err != nil {
+				return false, err
+			}
+			isBootableImage = podmanImgData.Labels["ostree.bootable"]
+		}
+	} else {
+		isBootableImage = imageData.Labels["ostree.bootable"]
+	}
+	// We may have pulled in OSContainer image as fallback during podmanCopy() or podmanInspect()
+	defer exec.Command("podman", "rmi", imgURL).Run()
+
+	return isBootableImage == "true", nil
+
+}
+
+// RebaseLayered rebases system or errors if already rebased
+func (r *RpmOstreeClient) RebaseLayered(imgURL string) (err error) {
+	glog.Infof("Executing rebase to %s", imgURL)
+	args := []string{"rebase", "--experimental", "ostree-unverified-registry:" + imgURL}
+
+	return runRpmOstree(args...)
```

Comment:
Isn't this simpler as 

```suggestion
	return runRpmOstree("rebase", "--experimental", "ostree-unverified-registry:" + imgURL)
```

Reply:
yep, simplified. Thank you.